src/modules/accounting/bunker.py

```diff
@@ -82,18 +80,25 @@ def is_bunker_mode(
             logger.info({"msg": "No one report yet. Bunker status will not be checked"})
             return False
 
-        self.all_validators, self.lido_keys, self.lido_validators = self._get_lido_validators_with_others(blockstamp)
+        self.all_validators = {
+            v.validator.pubkey: v for v in self.w3.cc.get_validators_no_cache(blockstamp.slot_number)
+        }
+        self.lido_keys = {str(k.key): k for k in self.w3.kac.get_all_lido_keys(blockstamp)}
+        self.lido_validators = {
+            v.validator.pubkey: v
+            for v in self.w3.lido_validators.get_lido_validators(blockstamp)
+        }
```

Comment:
since we often reuse different structs with keys or validators, like dicts, list, sets and so on, I would suggest separating them to some readable helpers and reuse them everywhere, respecting the naming convention